docs/Contributing/API-for-contributors.md

```diff
@@ -2539,6 +2540,33 @@ Downloads the Mobile Device Management (MDM) enrollment profile to install on th
 
 ---
 
+#### Migrate device to Fleet from another MDM solution
+
+Signals the Fleet server to send a webbook request with the device UUID and serial number to the webhook URL configured for MDM migration. **Requires Fleet Premium license**
+
+`POST /api/v1/fleet/device/{token}/migrate_mdm`
+
+##### Parameters
+
+| Name  | Type   | In   | Description                        |
+| ----- | ------ | ---- | ---------------------------------- |
+| token | string | path | The device's authentication token. |
+
+##### Example
+
+`POST /api/v1/fleet/device/abcdef012456789/migrate_mdm`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{}
+```
```

Comment:
nit: can we make this a `204` instead? You can do that by defining a method on the response:

```go
func (r deviceMigrateMDMResponse) Status() int  { return http.StatusNoContent }
```

Reply:
Sure thing! I actually had it as 204 originally and then reverted it when I was trying to debug another issue :)